.circleci/config.yml

```diff
@@ -0,0 +1,106 @@
+version: 2
+
+jobs:
+  build-client:
```

Comment:
@nikitasakov let's rename it like `bundle-client-dependencies` (or smth similar) because of `build` is about building ready for shipping, processed with babel (sometimes minimized), pure js files which usually keeps in _build_ dir